src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -1792,6 +1793,9 @@ private static MeshGeometry3D DrawTestMesh()
 
         internal void UpdateNearClipPlane()
         {
+
+            if (camera == null) return;
```

Comment:
If we pull the call from the OnSceneItemsChanged() the overall behaivor is better but we miss the call to update the clipping planes on the first run of the graph.  It may be better to leave that condition out as situations like run periodic, move a gizmo in the 3d view or testing cause some odd effects.  Then the null check is unnecessary.